Games/types/Mafia/roles/cards/Eavesdrop.js

```diff
@@ -0,0 +1,58 @@
+const Card = require("../../Card");
+const { PRIORITY_DAY_MEETING_DEFAULT } = require("../../const/Priority");
+const Message = require("../../../../core/Message");
+module.exports = class Eavesdrop extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Eavesdrop On": {
+                states: ["Day"],
+                flags: ["voting"],
+                action: {
+                    labels: ["eavesdrop"],
+                    priority: PRIORITY_DAY_MEETING_DEFAULT,
+                    run: function () {
+                        this.actor.role.data.stalk = this.target;
+                    }
+                }
+            },
+            "Eavesdropping": {
+                states: ["Night"],
+                flags: ["anonymous", "speech"],
+                inputType: "boolean",
+                leader: true,
+                canTalk: false,
+                action:{
+                    run: function(){
+                        delete this.actor.role.data.stalk;
+                    }
+                },
+                shouldMeet: function () {
+                    return this.data.stalk;
+                },
+            }
+        };
+        this.listeners = {
+            "message": function (message) {
+                if (this.game.getStateName() === "Night" &&
+                message.meeting &&
+                this.game.getMeetingByName("Eavesdropping") &&
+                message.meeting.hasJoined(this.data.stalk) &&
+                message.meeting.name !== "Eavesdropping"){
```

Comment:
maybe party chat should be intentionally excluded else the Eavesdrop meeting will just be the party chat spamming it

Reply:
I meant to make it only meetings that the Invisible man isn't in. I can update that